salt/modules/cmdmod.py

```diff
@@ -2653,16 +2654,25 @@ def shells():
     return ret
 
 
-def shell_info(shell):
+def shell_info(shell, **kwargs):
```

Comment:
If that is the case you have 633 module def's to go.  I think I would be the other way around, as your fixing the API's parameters location.  Salt needs to be more fluid than this and `**kwargs` allows it to be fluid.